pkg/api/validation/validation.go

```diff
@@ -4474,3 +4475,27 @@ func validateStorageNodeAffinityAnnotation(annotations map[string]string, fldPat
 	}
 	return policySpecified, allErrs
 }
+
+func validateSecurityContext(sc *api.SecurityContext) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if sc == nil {
+		return allErrs
+	}
+
+	if sc.AllowPrivilegeEscalation != nil && !*sc.AllowPrivilegeEscalation {
+		if *sc.Privileged {
```

Comment:
@jess, found the trace backs : http://paste.openstack.org/show/621629/

Looks like we need to check for nil here
```
if sc.Privileged != nil && *sc.Privileged
```

Reply:
omg you are my new favorite person on the planet